modules/ccs-gcp-customer-procedure.adoc

```diff
@@ -9,7 +9,13 @@
 The Customer Cloud Subscription (CCS) model allows Red{nbsp}Hat to deploy and manage {product-title} into a customer's Google Cloud Platform (GCP) project. Red Hat requires several prerequisites to provide these services.
```

Comment:
```suggestion
The Customer Cloud Subscription (CCS) model allows Red{nbsp}Hat to deploy and manage {product-title} into a customer's Google Cloud Platform (GCP) project. Red{nbsp}Hat requires several prerequisites to be completed before providing these services.
```

Know this is outside scope for this PR but this sentence doesn't make sense as is. Struggling with the wording a bit on this one - this is the best I could come up with.